db2/docker-compose.yaml

```diff
@@ -38,7 +38,7 @@ services:
 
   tdk:
     container_name: tdk
-    image: synthesizedio/synthesized-tdk-cli:latest
+    image: synthesizedio/synthesized-tdk-cli:v1.36.1
```

Comment:
Could we somehow centralize the version of the image, perhaps by using a single env file?

Reply:
Done